app/src/interfaces/useStandardsApi.interface.ts

```diff
@@ -0,0 +1,26 @@
+import {
+  CBQualitativeMeasurementTypeDefinition,
+  CBQuantitativeMeasurementTypeDefinition,
+  ICollectionUnitResponse
+} from './useCritterApi.interface';
+
+/**
+ * Data standards for a taxon
+ *
+ * @export
+ * @interface IGetSpeciesStandardsResponse
+ */
+export interface IGetSpeciesStandardsResponse {
+  tsn: number;
+  scientificName: string;
+  measurements: {
+    qualitative: CBQualitativeMeasurementTypeDefinition[];
+    quantitative: CBQuantitativeMeasurementTypeDefinition[];
+  };
+  markingBodyLocations: {
```

Comment:
Maybe preferrable to query without the "asSelect" format. To be more inline with what a `markingBodyLocation` actually is from critterbase.

Reply:
"asSelect" format condenses the object down to id, key, value vs the original object shape

Reply:
Body position just needs the label, but might update in the future as the standards page evolves